src/queue/Playlist.hxx

```diff
@@ -114,7 +114,7 @@ struct playlist {
 	int GetCurrentPosition() const noexcept;
 
 	gcc_pure
-	int GetNextPosition() const noexcept;
+	int GetNextPosition(unsigned index) const noexcept;
```

Comment:
Can we have `index=1` as default value, so existing callers don't need to be migrated?
Maybe rename the parameter to `offset`?